.eslintignore

```diff
@@ -1 +1,7 @@
 test/_fixtures
+test/datasource/**/_fixtures
+test/manager/**/_fixtures
+test/platform/**/_fixtures
+test/config/**/_fixtures
+test/workers/**/_fixtures
+test/logger/**/_fixtures
```

Comment:
This is fine if we need, but is there any simpler pattern that just lets us ignore any `_fixtures` directory no matter where it is?

Reply:
@rarkins Replaced with a glob that will match any `_fixtures` directory in test directory